k6/health.js

```diff
@@ -0,0 +1,10 @@
+import http from 'k6/http';
+import { sleep } from 'k6';
+export const options = {
+  vus: 10,
+  duration: '30s',
```

Comment:
Codacy has a fix for the issue: [Replace `'30s'` with `"30s"`](https://app.codacy.com/gh/dasch-swiss/dsp-api/pullRequest?prid=10006168)
```suggestion
  duration: "30s",
```